src/app/pages/form/form.component.html

```diff
@@ -1 +1,67 @@
-<p>The Form Component</p>
\ No newline at end of file
+<p-card
+  styleClass="user-form"
```

Comment:
jeśli potrzebujesz tej klasy to możesz dać to poniżej, nie musisz tego rozdzielać